jobs/component_jobs.groovy

```diff
@@ -57,23 +53,27 @@ repos.each { Map repo ->
             triggerPhrase('OK to test')
             orgWhitelist(['deis'])
             allowMembersOfWhitelistedOrgsAsAdmin()
-            // TODO: configure commit status messaging
-            // extensions {
-            //   commitStatus {
-            //     context('deploy to staging site')
-            //       triggeredStatus('starting deployment to staging site...')
-            //       startedStatus('deploying to staging site...')
-            //       statusUrl('http://mystatussite.com/prs')
-            //       completedStatus('SUCCESS', 'All is well')
-            //       completedStatus('FAILURE', 'Something went wrong. Investigate!')
-            //       completedStatus('PENDING', 'still in progress...')
-            //       completedStatus('ERROR', 'Something went really wrong. Investigate!')
-            //     }
-            //   }
+            extensions {
```

Comment:
Honestly not sure how this block is going to shake out -- this might take some trial/error on seeing how the commit status on a GH pull request behaves...